src/OpenTK.GLWidget/GLWidget.cs

```diff
@@ -15,6 +15,7 @@ namespace OpenTK
     /// <summary>
     /// The <see cref="GLWidget"/> is a GTK widget for which an OpenGL context can be used to draw arbitrary graphics.
     /// </summary>
+    [CLSCompliant(false)]
```

Comment:
Is this needed? Surely it's sufficient to just mark the non-compliant members, or the whole class. Not both.